apps/test-suite/tests/ScreenOrientation.js

```diff
@@ -1,50 +1,20 @@
 'use strict';
 
-import { ScreenOrientation } from 'expo';
+import * as ScreenOrientation from 'expo-screen-orientation';
 import { Platform } from 'react-native';
 
 export const name = 'ScreenOrientation';
 
-const convertToCoarseOrientation = orientation => {
```

Comment:
It was needed because we used traitCollection to determine the device orientation and it had no information of which landscape left-right and portrait up-down the device was in. Now we use UIDevice and it has that info, so afaik we can do it.